vulnerable_people_form/form_pages/shared/sms_validation.py

```diff
@@ -0,0 +1,48 @@
+# NOTIFY CODE
+
+OBSCURE_WHITESPACE = (
+    '\u180E'  # Mongolian vowel separator
+    '\u200B'  # zero width space
+    '\u200C'  # zero width non-joiner
+    '\u200D'  # zero width joiner
+    '\u2060'  # word joiner
+    '\u00A0'  # non breaking space
+    '\uFEFF'  # zero width non-breaking space
+)
+
+uk_prefix = '44'
+
+
+class InvalidPhoneError(Exception):
+
+    def __init__(self, message=None):
+        super().__init__(message or 'Not a valid phone number')
+
+
+def validate_uk_phone_number(number):
```

Comment:
I think this should be called something that indicates it is looking for mobile phone numbers only.

Reply:
I have renamed it `validate_notify_compatible_uk_mobile_number`